RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleMVACut.cc

```diff
@@ -11,16 +12,19 @@ class GsfEleMVACut : public CutApplicatorWithEventContentBase {
   void setConsumes(edm::ConsumesCollector&) final;
   void getEventContent(const edm::EventBase&) final;
 
-  double value(const reco::CandidatePtr& cand) const final;
-
   CandidateType candidateType() const final {
     return ELECTRON;
   }
 
 private:
 
-  // Cut values
-  const std::vector<double> mvaCutValues;
+  double value(const reco::CandidatePtr& cand) const final;
```

Comment:
Why ```double``` instead of ```float```?
It returns what is computed as ```float``` anyhow...

Reply:
It's what the template expects:
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/SelectorUtils/interface/CandidateCut.h#L14